Update snippets.yml

drjwbaker opened the pull request:
Add `/en/` preface to EN menu pages to stop hang/redirect per issue #826

I think I caught them all :)

Commit:
Update snippets.yml

Add `/en/` preface to EN menu pages to stop hang/redirect per issue #826

acrymble commented:
There's a problem with /en/lessons, which returns a 404 page. I must have done something wrong in my earlier pull request. @mdlincoln

drjwbaker commented:
Ack.

mdlincoln committed:
correct pathing issues in en/lessons

The permalink field sets the path that will resolve to this page. And
for some reason the redirect_from only worked if there was a trailing
slash.

mdlincoln commented:
Turns out the solution lay in the `permalink` field, which we use rarely in this site. The remaining 404s are being repaired in #859
